Example/GiniVision/ComponentAPIReviewViewController.swift

```diff
@@ -22,19 +22,19 @@ class ComponentAPIReviewViewController: UIViewController {
     /**
      The image data of the captured document to be reviewed.
      */
-    var imageData: Data!
+    var document: GiniVisionDocument!
     
-    fileprivate var originalData: Data?
+    fileprivate var originalDocument: GiniVisionDocument?
```

Comment:
And here... the original is an optional but the output is force unwrapped. I'd really like to unify it.

Reply:
Me too, but as commented above I think that it is not possible since we are using `Storyboards`. Or is there other way?

Reply:
Well, you can start with an empty document. That's the alternative. But it's not ideal. Ok, lets keep it.